enhancements/update/synchronized-upgrades.md

```diff
@@ -0,0 +1,223 @@
+---
+title: Synchronized Upgrades Between Clusters
+authors:
+  - "@danwinship"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+api-approvers: # in case of new or modified APIs or API extensions (CRDs, aggregated apiservers, webhooks, finalizers)
+  - TBD
+creation-date: 2021-01-11
+last-updated: 2021-01-11
+tracking-link:
+  - https://issues.redhat.com/browse/SDN-2603
+see-also:
+  - "/enhancements/network/dpu/overview.md"
+---
+
+# Synchronized Upgrades Between Clusters
+
+## Summary
+
+In a [cluster with DPUs](../network/dpu/overview.md) (eg, BlueField-2
+NICs), the x86 hosts form one OCP cluster, and the DPU ARM systems
+form a second OCP cluster. This makes upgrades to new OCP releases
+complicated, because there is currently no way to synchronize upgrades
+between the two clusters, but rebooting the BF-2 systems as part of
+the MCO upgrade will cause a network outage on the x86 systems. In
+order for upgrades to work smoothly, we need to synchronize the
+reboots between the two clusters, so that the BF-2 systems are only
+rebooted when their corresponding x86 hosts have been cordoned and
+drained.
+
+Please refer to the "Glossary" section of the [DPU Overview
+Enhancement](../network/dpu/overview.md).
+
+## Motivation
+
+### Goals
+
+- Make upgrades work smoothly in clusters running with DPU support, by
+  synchronizing the reboots of nodes between the infra cluster and the
+  tenant cluster.
+
+### Non-Goals
+
+- Supporting synchronized upgrades of more than 2 clusters at once.
+
+## Proposal
+
+### User Stories
+
+As the administrator of a cluster using DPUs, I want to be able to do
+z-stream upgrades without causing unnecessary network outages.
```

Comment:
No, this applies to y-stream upgrades too. It's just that you might be able to convince people that y-stream upgrades are allowed to have a little bit of disruption since they're rare, but you can't make that argument for z-stream upgrades.